mlir/test/Dialect/Linalg/generalize-tensor-pack.mlir

```diff
@@ -1,5 +1,4 @@
-// RUN: mlir-opt -split-input-file --test-linalg-transform-patterns="test-generalize-tensor-pack"  %s | FileCheck %s
-
+// RUN: mlir-opt  --transform-preload-library='transform-library-paths=%p/td/generalize_pack.mlir' -split-input-file  --transform-interpreter --test-transform-dialect-erase-schedule %s | FileCheck %s
```

Comment:
I'm not familiar with how `transform-preload-library` works in this case. Do we need the `--test-transform-dialect-erase-schedule` flag?

Reply:
> Do we need the --test-transform-dialect-erase-schedule flag?

We do not - old habits :) I will remove it before landing.